### PR TITLE
METRON-1145: Profiler mpack does not create kafka acl and hbase acls correctly in kerberized cluster

### PR DESCRIPTION
**Pull request by cestella**
## Contributor Comments
Title says it all. The management pack will not start the profiler in a kerberized cluster due to us not setting the acls correctly for the hbase tables and kafka group acls.

To test this, spin up full-dev, [kerberize it](https://github.com/apache/metron/blob/master/metron-deployment/Kerberos-ambari-setup.md) and ensure the profiler topology starts without error.

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel). 
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [x] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [x] Have you included steps or a guide to how the change may be verified and tested manually?
- [x] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
  ```
  mvn -q clean integration-test install && build_utils/verify_licenses.sh 
  ```

- [x] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

### CODE REVIEW

**Comment by ottobackwards** [comment]
You should add something to the PR Template
 [ ] Does this require ACL's  

or something

**Comment by cestella** [comment]
haha actually, it's more like, "if a mpack, have you tested on kerberized full-dev"

**Comment by merrimanr** [comment]
Tested this in full dev with Kerberos enabled.  The profiler topology starts up without issue and data appears in HBase after creating a profile.  +1

File: metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/profiler_master.py [hunk 1]
```diff
@@ -59,6 +60,9 @@ def configure(self, env, upgrade_type=None, config_dir=None):
             commands.create_hbase_tables()
         if params.security_enabled and not commands.is_hbase_acl_configured():
             commands.set_hbase_acls()
```
Comment (nickwallen) [inline metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/profiler_master.py]: Is there a reason that we do not call `commands.set_hbase_acl_configured()` here?

Comment (nickwallen) [reply metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/profiler_master.py]: Sorry to bring this up on an old, closed PR, but I am currently trying to track a bug that is giving me an uber headache. :)  Thought this was the easiest way to reach out.

Comment (justinleet) [reply metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/profiler_master.py]: @nickwallen It happens at the end of `set_hbase_acls()`

Comment (nickwallen) [reply metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/profiler_master.py]: Ah, daft, you're right.  It works different than the Kafka ACL functions.

**Pull request closed without merging**